server/cars.js

```diff
@@ -1,11 +1,14 @@
 const mongoose = require('mongoose');
 const express = require('express');
+const moment = require('moment');
 const router = express.Router();
 const auth = require('./auth.js');
 
 const users = require('./users.js');
 const User = users.model;
 
+const { intervals } = require('./constants')
```

Comment:
Importing the values for how often a service should be done so it's not hard coded